tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -35,11 +36,55 @@
 _doc_root: str = ""
 
 
+class CapsuleType(IntEnum):
+    # Defined in
+    # https://www.ietf.org/archive/id/draft-ietf-masque-h3-datagram-03.html.
+    DATAGRAM = 0xff37a0
+    REGISTER_DATAGRAM_CONTEXT = 0xff37a1
+    REGISTER_DATAGRAM_NO_CONTEXT = 0xff37a2
+    CLOSE_DATAGRAM_CONTEXT = 0xff37a3
```

Comment:
Do we expect to use these in the near future? If not, maybe it's better to add them when we actually use these.

Reply:
OK, removing unused variables won't hurt...